js/build_map.js

```diff
@@ -33,5 +36,6 @@ function create_map() {
 	var state_data = arguments[1];
   
   add_states(map, state_data);
+  add_timeslider(map, years, chart_width, chart_width);
```

Comment:
wait ... I think I just solved my slider placement issue. I was passing chart_width in twice so it was using the wrong value for chart_height inside this function. Fix on its way!